components/button/test/floating-buttons.html

```diff
@@ -1,191 +0,0 @@
-<!doctype html>
```

Comment:
I deleted these tests as they had a huge amount of overlap with the visual diff tests and seemed unnecessary.